source/includes/_resource.customers.md

```diff
@@ -139,6 +164,8 @@ $ curl -X GET "https://api.vhx.tv/customers/1" \
 }
 ```
 
+> <div class="highlight shell text"><p>the _embedded object contains a browse_url that can be used to authenticate a customer session.</p></div>
```

Comment:
@kaitlinwiener this is the line I added about the browse_url. Let me know what you think.

Reply:
great!